registry/registry-consul/src/test/java/com/alipay/sofa/rpc/registry/consul/ConsulRegistryAclTest.java

```diff
@@ -0,0 +1,405 @@
+ */
+public class ConsulRegistryAclTest {
+
+    private static final String INTERFACE_ID        = "com.alipay.sofa.rpc.registry.consul.TestService";
+
+    private static final String CONSUL_SERVICE_NAME = "test-service";
+
+    private ConsulProcess       consul;
+
+    private RegistryConfig      registryConfig;
+
+    private String              token               = "db4175d7-c4bf-46c2-a071-8d4fd80c30fc2022-02-20";
+
+    private ConsulRegistry      registry;
+
+    @Before
+    public void setup() {
+        consul = ConsulStarterBuilder.consulStarter()
+            .withConsulVersion("1.4.0")
+            .withToken(token)
+            .withCustomConfig("{\n" +
+                "          \"acl\": {\n" +
+                "            \"enabled\": true,\n" +
+                "            \"default_policy\": \"deny\",\n" +
+                "            \"down_policy\": \"deny\",\n" +
+                "            \"tokens\": {\n" +
+                "                \"agent\": \"db4175d7-c4bf-46c2-a071-8d4fd80c30fc2022-02-20\",\n" +
+                "                \"master\": \"db4175d7-c4bf-46c2-a071-8d4fd80c30fc2022-02-20\"\n" +
+                "            }\n" +
+                "        }")
+            .build()
+            .start();
+
+        registryConfig = new RegistryConfig()
+            .setProtocol("consul")
+            .setAddress("127.0.0.1:" + consul.getHttpPort())
+            .setParameter("token", token)
+            .setRegister(true);
+
+        registry = (ConsulRegistry) RegistryFactory.getRegistry(registryConfig);
+        registry.init();
+    }
+
+    @After
+    public void tearDown() {
+        registry.destroy();
+        consul.close();
+        registry = null;
+    }
+
+    @Test
+    public void testRegister() {
+        ProviderConfig<?> providerConfig = providerConfig("consul-test-1", 12200, 12201, 12202);
+        registry.register(providerConfig);
+
+        ConsulClient consulClient = new ConsulClient("localhost", consul.getHttpPort());
+        HealthServicesRequest request = HealthServicesRequest.newBuilder().setPassing(true).setToken(token).build();
+        assertUntil(() -> {
+            Response<List<HealthService>> healthServices = consulClient.getHealthServices(INTERFACE_ID, request);
+            Assert.assertEquals(3, healthServices.getValue().size());
+        }, 10, TimeUnit.SECONDS);
+
+        registry.unRegister(providerConfig);
+
+        assertUntil(() -> {
+            Response<List<HealthService>> healthServices = consulClient.getHealthServices(INTERFACE_ID, request);
+            Assert.assertEquals(0, healthServices.getValue().size());
+        }, 10, TimeUnit.SECONDS);
+    }
+
+    @Test
+    public void testRegisterWithoutToken() {
+        ProviderConfig<?> providerConfig = providerConfig("consul-test-1", 12200, 12201, 12202);
+        registry.register(providerConfig);
+
+        ConsulClient consulClient = new ConsulClient("localhost", consul.getHttpPort());
```

Comment:
There are some places in this piece that are repeated with `testRegister`.Is there any other intention to explain, if not, can delete it or delete `testRegister`. The following withoutToken tests have the same problem.

Reply:
I want to make sure that my commit still works on miss token, but it looks like  `ConsulRegistryTest` job,not `ConsulRegistryAclTest`.
I will delete some code  like `testRegister`.